escrow/escrow.go

```diff
@@ -45,8 +45,18 @@ func (h *lockingPaymentChannelService) PaymentChannel(key *PaymentChannelKey) (c
 		return
 	}
 
+
 	blockchainChannel, blockchainOk, err := h.blockchainReader.GetChannelStateFromBlockchain(key)
+
 	if !storageOk {
+		//Group ID check is only done for the first time , when the channel is added to storage from the block chain ,
+		//if the channel is already present in the storage the group ID check is skipped.
+	    if blockchainChannel != nil {
+			blockChainGroupID,err := h.blockchainReader.replicaGroupID()
```

Comment:
As `replicaGroupId` is not used in blockchain reader anymore I would move it into `lockingPaymentChannelService` itself. But it will probably require fixing unit tests, so it makes sense to do separate PR for this.

Reply:
Will submit a separate PR by tomorrow , hopefully by then this is approved :)